pkg/cmd/urfave/urfave.go

```diff
@@ -101,9 +111,17 @@ func GetTraceeRunner(c *cli.Context, version string) (cmd.Runner, error) {
 
 	// Container information printer flag
 
-	printerConfig := output.PrinterConfig
-	printerConfig.ContainerMode = cmd.GetContainerMode(cfg)
-	cfg.Output = output.TraceeConfig
+	printers := make([]printer.EventPrinter, 0, len(output.PrinterConfigs))
+	for _, pConfig := range output.PrinterConfigs {
+		pConfig.ContainerMode = getContainerMode(cfg)
```

Comment:
I think you can use here the `(fs *FilterScopes) ContainerFilterEnabled()` helper.

Reply:
@geyslan The `getContainerMode`is using `ContainerFilterEnabled()`helper, right?

Reply:
Actually you're using the FilterScope helper, I've mentioned the FilterScope**S** one, which already provides the needed information based on a bitmask filled through (fs *FilterScopes) Compute().